Source/UI/NewUI/PaymentForm/View/SecurityCodeView.swift

```diff
@@ -11,28 +11,22 @@ import Checkout
 public final class SecurityCodeView: UIView {
 
   private let securityCodeCount = 4
-
-  private let environment: Environment
+  private var cardValidator: CardValidator
   private(set) var style: CellTextFieldStyle?
 
-  private lazy var cardValidator: CardValidator = {
-    CardValidator(environment: environment.checkoutEnvironment)
-  }()
-
-  private lazy var dateInputView: InputView = {
+  private lazy var codeInputView: InputView = {
```

Comment:
👍

Reply:
I am not sure, but is `InputView` automatically securing the leaking of the input from user?

Reply:
Found it, there is a hierarchy of InputView -> TextFieldContainer (UIView) -> TextFieldView -> SecureDisplayView -> UITextField 👍